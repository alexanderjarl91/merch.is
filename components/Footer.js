import React from "react";
import styles from "../styles/Footer.module.css";
import { IoLogoInstagram } from "react-icons/io";
import { FaFacebook } from "react-icons/fa";
import Link from "next/link";

export default function Footer() {
  return (
    <div className={styles.footer}>
      <div className="content">
        <a href="www.instagram.com">
          <IoLogoInstagram className={styles.social} />{" "}
        </a>
        <a href="www.facebook.com">
          <FaFacebook className={styles.social} />
        </a>
        <p className={styles.skilmalar}>Skilm√°lar</p>
        <Link href="/">copyright merch.is 2021</Link>
        {/* <button
          onClick={() => {
            console.log(currentUser);
          }}
        >
          currentUser
        </button>
        <button
          onClick={() => {
            console.log(users);
          }}
        >
          users
        </button>
        <button
          onClick={() => {
            getUsers();
          }}
        >
          getUserData()
        </button>

        <button
          onClick={() => {
            handleLogin();
          }}
        >
          login
        </button>
        <button
          onClick={() => {
            handleLogout();
          }}
        >
          log out
        </button> */}
      </div>
    </div>
  );
}
