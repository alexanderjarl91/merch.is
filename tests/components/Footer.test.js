import { shallow } from "enzyme";
import React from "react";
import Footer from "../../components/Footer";

it("expect Footer component to match Snapshot", () => {
  expect(shallow(<Footer />)).toMatchSnapshot();
});

it("expect to render component Footer", () => {
  expect(shallow(<Footer />).length).toEqual(1);
});
