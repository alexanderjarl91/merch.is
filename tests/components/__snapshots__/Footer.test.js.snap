// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect Footer component to match Snapshot 1`] = `ShallowWrapper {}`;
